hydro_deploy/core/src/ssh.rs

```diff
@@ -75,81 +80,85 @@ impl LaunchedBinary for LaunchedSshBinary {
         }
     }
 
-    async fn wait(&mut self) -> Option<i32> {
+    async fn wait(&mut self) -> Result<i32> {
         self.channel.wait_eof().await.unwrap();
-        let ret = self.exit_code();
+        let exit_code = self.channel.exit_status()?;
         self.channel.wait_close().await.unwrap();
-        ret
+
+        Ok(exit_code)
+    }
+
+    async fn stop(&mut self) -> Result<()> {
```

Comment:
Should we cal this function `ctrl_c`? So that it is clear that this will non-gracefully end the process.

Reply:
I documented it as `/// If the process is still running, force stop it, then run post-run tasks.`
So it won't necessarily always do a ctrl_c or a kill, it'll only do that if the process hasn't been `wait()`ed for already and is still running

Reply:
Updated to `/// If the process is still running, force stop it. Then always run post-run tasks.`